[BULK] - DocuTune - Rebranding of Azure Active Directory to Microsoft Entra terminology (#36)
MicrosoftDocs/microsoft-authentication-library-javascript: msal-javascript-conceptual/browser/working-with-b2c.md

}
```

<a name='aad-vs-b2c-endpoints'></a>

## Microsoft Entra vs B2C Endpoints

> A major difference between **Microsoft Entra ID** (*Microsoft Entra ID* for short) vs. **B2C** tenants are with respect to their **endpoints**.

Reviewer: ```suggestion
A major difference between **Microsoft Entra ID**  and **Azure AD B2C** tenants is their **endpoints**.
```
